libraries/AP_Frsky_Telem/AP_Frsky_Backend.h

```diff
@@ -58,6 +58,7 @@ class AP_Frsky_Backend
 
     void calc_nav_alt(void);
     void calc_gps_position(void);
+    bool calc_rpm(const uint8_t instance, int32_t &value) const;
```

Comment:
For this sort of thing it's worht adding either a `WARN_IF_UNUSED` or a note that the passed-in-parameter is not modified if the function returns false.